static/webview.html

```diff
@@ -190,23 +158,63 @@
       let node = g.append("g").attr("class", "nodes").selectAll(".node");
       let text = g.append("g").attr("class", "text").selectAll(".text");
 
-      const zoomActions = () => {
-        const scale = d3.event.transform;
-        zoomLevel = scale.k;
-        g.attr("transform", scale);
+      const resize = () => {
+        if (d3.event) {
+          const scale = d3.event.transform;
+          zoomLevel = scale.k;
+          g.attr("transform", scale);
+        }
 
-        const zoomOrKeep = (value) => (scale.k >= 1 ? value / scale.k : value);
+        const zoomOrKeep = (value) => (zoomLevel >= 1 ? value / zoomLevel : value);
 
         const font = Math.max(Math.round(zoomOrKeep(FONT_SIZE)), 1);
 
         text.attr("font-size", `${font}px`);
         text.attr("y", (d) => d.y - zoomOrKeep(FONT_BASELINE));
         link.attr("stroke-width", zoomOrKeep(STROKE));
         node.attr("r", zoomOrKeep(RADIUS));
+        svg.selectAll('circle').filter(function() {
+          return d3.select(this).attr("active");
+        }).attr("r", zoomOrKeep(ACTIVE_RADIUS));
```

Comment:
I know this is not covered in CI yet, but please stick to Prettier formatting rules which gives:

```js
svg
  .selectAll('circle')
  .filter(function() {
    return d3.select(this).attr('active');
  })
  .attr('r', zoomOrKeep(ACTIVE_RADIUS));
```

BTW, can it be done in a way to avoid using `function` and `this`? IIRC this callback is given a param (usually called `d`) which contains some information. Maybe `active` can be extracted from it too?

Reply:
Hi @tchayen thanks a lot for taking a look!

I have updated it to use a function accessing the DOM element via the callback params like you mentioned.

I think that `.attr(...)` is probably still the best way to access the `active` attribute.